sdk/go/common/env/env.go

```diff
@@ -100,6 +102,10 @@ var SuppressCopilotLink = env.Bool("SUPPRESS_COPILOT_LINK",
 var FallbackToStateSecretsManager = env.Bool("FALLBACK_TO_STATE_SECRETS_MANAGER",
 	"Use the snapshot secrets manager as a fallback when the stack configuration is missing or incomplete.")
 
+// but the NumCPU function returns an int.
```

Comment:
stray comment

Reply:
Thank you. Yep, left from trying to calculate in env.go.